tests/e2e/configurer/upgrade.go

```diff
@@ -143,17 +143,7 @@ func (uc *UpgradeConfigurer) CreatePreUpgradeState() error {
 
 	v17SuperfluidAssets := v17GetSuperfluidAssets()
 
-	wg.Add(4)
-
-	go func() {
-		defer wg.Done()
-		chainANode.FundCommunityPool(initialization.ValidatorWalletName, strAllUpgradeBaseDenoms())
-	}()
-
-	go func() {
-		defer wg.Done()
-		chainBNode.FundCommunityPool(initialization.ValidatorWalletName, strAllUpgradeBaseDenoms())
-	}()
```

Comment:
Note to reviewer, we no longer need to fund the community pool with every baseAsset because we now swap osmo from the community pool for the baseAsset!